### ISSUE TITLE
Re: Failed to Create Filtered Deck

### ISSUE THREAD

**Issue by HasanBM** [fonol/anki-search-inside-add-card#245]
When I reviewed using the SIAC add-on first time, it created a filtered deck. I (foolishly) deleted this, as I wanted the cards to be in my main deck for this topic.

Unfortunately, despite uninstalling, deleting the files in the locations Anki stores them, as well as removing the SIAC folder in my hard drive, I have not been able to get rid of the "Could Not Create Filtered Deck" error that keeps coming up.

![Screen Shot 2021-03-19 at 12 45 40](https://user-images.githubusercontent.com/68597089/111782996-a09a7c80-88b1-11eb-9dfe-29530630e946.jpg)
![Screen Shot 2021-03-19 at 12 45 59](https://user-images.githubusercontent.com/68597089/111783016-a5f7c700-88b1-11eb-8993-dd9b73415059.jpg)

Unfortunately, without this filtered deck, the add-on is fairly pointless. Is there any way to get this functionality back?

**Comment by fonol** [comment]
Hi, 
hard to pinpoint what's going wrong.
If you run Anki from the console, do you see any errors printed there when it shows the "Failed to create filtered deck" message?
What Anki version are you on?

Unrelated, but I see some visual bugs on your screenshots: There are some calendar icons floating around in weird places, 
does this always happen for you? If so, does it also occur with other add-ons disabled?

**Comment by HasanBM** [comment]
Hi,

Not sure what you mean by the Calendar icon. I am a CLI noob so I'm not sure how to run Anki from the command line!
Also, the error repeats with the other add-on I have enabled - that is the Image Occlusion Enhanced 21 add-on.

I created another profile with the same add-ons, and uploaded the same PDFs. This time, the error did not show until I synced my original account. So, a workaround might be: create another profile and AnkiWeb account, then sign in on that. Don't download the cards from my original profile, and continue to use the original profile until the 'interrupt' signal goes off on SIAC. Switch profiles, then study the text, then switch profiles again, and continue revising.

Not the quickest or the easiest, but it might work!

**Comment by fonol** [comment]
> Not sure what you mean by the Calendar icon.

![image](https://user-images.githubusercontent.com/22644769/111898285-5a662a00-8a25-11eb-9566-8cb42af6c60f.png)
I meant these, but that is fixed in the new version.

> I created another profile with the same add-ons, and uploaded the same PDFs. This time, the error did not show until I synced my original account. So, a workaround might be: create another profile and AnkiWeb account, then sign in on that. Don't download the cards from my original profile, and continue to use the original profile until the 'interrupt' signal goes off on SIAC. Switch profiles, then study the text, then switch profiles again, and continue revising.
> 
> Not the quickest or the easiest, but it might work!

Hm, but it should work without all that hassle! 

> I am a CLI noob so I'm not sure how to run Anki from the command line!

With the new update on Ankiweb, the add-on should include the error message in the tooltip (where it says "Failed to create filtered deck"). So maybe you could update,  try again, and screenshot/copy what error message it says in the tooltip. That would be helpful.

**Comment by HasanBM** [comment]
> > Not sure what you mean by the Calendar icon.
> 
> ![image](https://user-images.githubusercontent.com/22644769/111898285-5a662a00-8a25-11eb-9566-8cb42af6c60f.png)
> I meant these, but that is fixed in the new version.
> 
> > I created another profile with the same add-ons, and uploaded the same PDFs. This time, the error did not show until I synced my original account. So, a workaround might be: create another profile and AnkiWeb account, then sign in on that. Don't download the cards from my original profile, and continue to use the original profile until the 'interrupt' signal goes off on SIAC. Switch profiles, then study the text, then switch profiles again, and continue revising.
> > Not the quickest or the easiest, but it might work!
> 
> Hm, but it should work without all that hassle!

Agreed - I think the issue may have to do with backups downloading and therefore overriding the deck SIAC creates in my desktop app profile. The issue does not occur when I run the second profile, as I've not deleted the deck there.


> With the new update on Ankiweb, the add-on should include the error message in the tooltip (where it says "Failed to create filtered deck"). So maybe you could update, try again, and screenshot/copy what error message it says in the tooltip. That would be helpful.

Attached.
![Screen Shot 2021-03-21 at 11 14 26](https://user-images.githubusercontent.com/68597089/111902767-e5035500-8a36-11eb-8621-9767e683e366.jpg)

**Comment by fonol** [comment]
>  The issue does not occur when I run the second profile, as I've not deleted the deck there.

As it is intended, you should totally be able to delete the filtered deck, it should then simply recreate it again.

> Attached.

Thanks, that gives me a clue. I might have a fix or not with the next update.

**Comment by OcileCollection** [comment]
@HasanBM Have you tried the new update? Did it solve the issue?
